lib/lotus/validations/errors.rb

```diff
@@ -68,20 +70,7 @@ def clear
       #
       # @since 0.1.0
       def each(&blk)
-        errors.each(&blk)
-      end
-
-      # Iterate thru the errors, yields the given block and collect the
-      # returning value.
-      #
-      # @param blk [Proc] the given block
-      # @yield [error] a Lotus::Validations::Error
-      #
-      # @see Lotus::Validations::Error
-      #
-      # @since 0.1.0
-      def map(&blk)
```

Comment:
Since we have it as `Enumerable` we don't need to implement `map`, `each` is enough.